### PR TITLE
chore: add C3 to the bug issue template

### PR DESCRIPTION
**Pull request by petebacondarwin**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-template.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ body:
       label: Which Cloudflare product(s) does this pertain to?
       multiple: true
       options:
+        - C3
```
Comment (IgorMinar) [inline .github/ISSUE_TEMPLATE/bug-template.yaml:20]: can you add the same to the feature request template as well? thanks!

Comment (petebacondarwin) [reply .github/ISSUE_TEMPLATE/bug-template.yaml:20]: The feature request template does not have a drop down.

Comment (petebacondarwin) [reply .github/ISSUE_TEMPLATE/bug-template.yaml:20]: image

Comment (IgorMinar) [reply .github/ISSUE_TEMPLATE/bug-template.yaml:20]: ah. bad assumption on my part. my apologies.

**Review by IgorMinar — APPROVED** [review]

**Pull request merged**